static/src/javascripts/projects/common/modules/identity/api.js

```diff
@@ -321,3 +321,10 @@ export const ajaxSignIn = (credentials: PasswordCredential) => {
         credentials: 'include',
     });
 };
+
+export const getUserData = (): Promise<Response> =>
```

Comment:
Used to infer if a user is logged in (response is OK), since the present of `GU_U` cookie does not necessarily imply that they are logged in.

Reply:
I'd rather not, since `getUserFromApi` relies on a callback rather than just returning a value. As a compromise could I keep the new function private to the file in which it's being used?